### DISCUSSION TITLE
How to interact with router/switches/firewall from the operational mode using netmiko?

### DISCUSSION THREAD

**Discussion by Birajan2019** [ktbyers/netmiko#2998] [General]
Hello,

I need to be able to login to a Juniper router, run an operational command that requests for a passsword, enter the password and finally print the prompt.

This is what I mean:

charlie@test-router> start shell user root
Password:
root@test-router:/var/home/su #

Any suggestion how we can do this with netmiko?

Thanks,
B

**Comment by ktbyers** [comment]
Something like this should work:

https://github.com/ktbyers/netmiko/blob/develop/EXAMPLES.md#handling-commands-that-prompt-timing

**Comment by Olfasalem** [comment]
Exercice netmiko pour configuration 3 router
